src/baseball/main.py

```
@@ -1,8 +1,87 @@
+
 def main():
-    """
-    프로그램의 진입점 함수.
-    여기에서 전체 프로그램 로직을 시작합니다.
-    """
+    print('숫자 야구 게임을 시작합니다')
+    while 1:
+
+        strike=0
+        ball=0
+        l=[]
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)